src/bes/lims/reports/forms/analyses_results.py

```diff
@@ -55,66 +58,59 @@ def process_form(self):
         # Add the first row (header)
         rows = [[
             _("Sample ID"),
-            _("Patient Name"),
-            _("Patient Surname"),
-            _("Patient Other Names"),
-            _("Patient Hospital #"),
-            _("Patient Date of Birth"),
+            _("Sample Type"),
             _("Patient Age"),
             _("Patient Gender"),
-            _("Test Date Collected"),
-            _("Test Date Tested"),
-            _("Test Category"),
+            _("Test Date and Time Collected"),
+            _("Test Date and Time Tested"),
+            _("Test Date and Time Verified"),
             _("Test Department"),
-            _("Test ID"),
+            _("Test Panels"),
             _("Test Type"),
             _("Test Result"),
+            _("Test Unit"),
             _("Site"),
-            _("Requesting Physician"),
         ]]
 
         # Add the info per analysis in a row
         for analysis in analyses:
             sample = analysis.getRequest()
-            fullname = sample.getField("PatientFullName").get(sample) or {}
             sampled = sample.getDateSampled()
             dob = sample.getDateOfBirth()[0]
             age = self.get_age(dob, sampled)
-            dob = self.parse_date_to_output(dob)
             sampled = self.parse_date_to_output(sampled)
             result_captured = self.parse_date_to_output(
                 analysis.getResultCaptureDate()
             )
-
+            result_verified = self.parse_date_to_output(
+                analysis.getDateVerified()
+            )
             # Only show results that appear on the final reports
             result = ""
             if analysis.getDatePublished():
                 result = analysis.getFormattedResult(html=False) or result
                 result = self.replace_html_breaklines(result)
 
             department = analysis.getDepartmentTitle() or ""
-            category = analysis.getCategoryTitle() or ""
+            panels = self.get_panels(analysis) or ""
```

Comment:
@nnpvaan note that "test panel" refers to the analysis profile assigned to a sample not the AST panel (https://github.com/beyondessential/bes.lims/issues/61#issuecomment-3044081340)